parser.cc

```diff
@@ -1067,7 +1067,9 @@ class TraceExpr : public Expr
 		                con.position.col);
 	}
 public:
+#ifdef DEBUG_PARSING
```

Comment:
This looks wrong, you should be able to use `TraceExpr` even when Pegmatite is built without `DEBUG_PARSING`.